appserver/pom.xml

```diff
@@ -106,7 +105,7 @@
         <com.ibm.jbatch.container.version>1.0.2</com.ibm.jbatch.container.version>
         <com.ibm.jbatch.spi.version>1.0.2</com.ibm.jbatch.spi.version>
         <javax.management.j2ee-api.version>1.1.3</javax.management.j2ee-api.version>
-	    <jboss.classfilewriter.version>1.2.1.Final</jboss.classfilewriter.version>
+	<jboss.classfilewriter.version>1.2.1.Final</jboss.classfilewriter.version>
```

Comment:
The extra add lines appear to be due to the indentation being filled in with tabs rather than spaces.

Reply:
Corrected